test/asset/asset-registry.test.mjs

```diff
@@ -178,6 +185,15 @@ describe('AssetRegistry', function () {
             });
         });
 
+        it('supports retry loading of container assets', function (done) {
+            sinon.spy(http, 'request');
```

Comment:
did you mean to leave sinon in here?

Reply:
yes, so it can assert on call count

Reply:
I see, thanks. @willeastcott just explained to me that sinon populates `http.request.callCount` below.